box/boxnavigator.py

```diff
@@ -149,6 +154,26 @@ def determine_direction_to_target(self, current_target: Pt) -> str:
 
         return dominant_direction
 
+    def rotation_anchor(self, current_target: Pt, current_box: Box) -> [Pt]:
+        width_half = current_box.width / 2
+        height_half = current_box.height / 2
+
+        if self.dominant_direction in ["left", "right"]:
+            # Calculate the distance from the center to the left and right sides of the box
+            anchor_1 = Pt(current_target.x, current_target.y - height_half)
+            anchor_2 = Pt(current_target.x, current_target.y + height_half)
+
+        elif self.dominant_direction in ["up", "down"]:
+            # Calculate the distance from the center to the top and bottom sides of the box
+            anchor_1 = Pt(current_target.x - width_half, current_target.y)
+            anchor_2 = Pt(current_target.x + width_half, current_target.y)
+        else:
```

Comment:
```suggestion
            anchor_2 = Pt(current_target.x + width_half, current_target.y)

        else:
```

Added spacing for consistency.